website/blog/2024-04-15-0.74-post.md

```diff
@@ -0,0 +1,407 @@
+
+</td>
+
+</tr>
+
+</table>
+
+:::
+
+## Support for `align-content: 'space-evenly'`
+
+[`alignContent: 'space-evenly'`](https://developer.mozilla.org/en-US/docs/Web/CSS/align-content#space-evenly) distributes the lines in a multi-line flex container using evenly spaced gaps, placed between line and container edges.
+
+<a href="https://www.w3.org/TR/css-align-3/#distribution-values">
+![alignContent references](../static/blog/assets/0.74-align-content.png)
+</a>
+
+## New Architecture support for `position: 'static'`
+
+Elements marked as [`position: 'static'`](https://developer.mozilla.org/en-US/docs/Web/CSS/position#static) may not be offset, and are not considered when determining the [containing block](https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block) of an absolutely positioned element. This allows positioning an element relative to an ancestor which is not its direct parent.
+
+<table>
+
+<tr>
+
+<td width="350">
+
+```jsx
+<View
+  style={{
+    backgroundColor: 'blue',
+    width: 200,
+    height: 200,
+    flexDirection: 'row-reverse',
+  }}>
+  <View
+    style={{
+      backgroundColor: 'red',
+      width: 100,
+      height: 100,
+      position: 'static',
+    }}>
+    <View
+      style={{
+        backgroundColor: 'green',
+        width: 25,
+        height: '25%',
+        left: 25,
+        top: 25,
+        position: 'absolute',
+      }}
+    />
+  </View>
+</View>
+```
+
+</td>
+
+<td width="300">
+
+![Static Example](../static/blog/assets/0.74-static-example.png)
+
+</td>
+
+</tr>
+
+</table>
+Notice how the green `View` declares `left` and `top` and it is positioned relative to the blue `View`, not its parent.
+
+`position: 'static'` may also affect [z-ordering](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context) of views. Similar to how `position: 'static'` works with containing blocks, static will not be considered when forming stacking contexts. This allows for stacking an element relative to an ancestor that is not a direct parent.
+
+<table>
+
+<tr>
+
+<td width="350">
+
+```jsx
+<View
+  style={{
+    backgroundColor: 'blue',
+    width: 200,
+    height: 200,
+    flexDirection: 'row',
+  }}>
+  <View
+    style={{
+      backgroundColor: 'orange',
+      width: 50,
+      height: 50,
+      left: 10,
+      position: 'relative',
+      zIndex: 10,
+    }}
+  />
+  <View
+    style={{
+      backgroundColor: 'red',
+      width: 50,
+      height: 50,
+      position: 'static',
+    }}>
+    <View
+      style={{
+        backgroundColor: 'green',
+        width: 25,
+        height: 25,
+        zIndex: 11,
+        position: 'absolute',
+      }}
+    />
+  </View>
+</View>
+```
+
+</td>
+
+<td width="300">
+
+![Static zIndex Example](../static/blog/assets/0.74-zindex-example.png)
+
+</td>
+
+</tr>
+
+</table>
+
+React Native continues to default to `position: 'relative'` when no `position` is set.
```

Comment:
I would probably remove this from the blogpost and create a working group post with the same content for the New Architecture, to be consistent with our communication strategy for the New Architecture.
Thoughts on this?

Reply:
It depends on how much content we have to highlight for 0.74.
If we don't have any headlines, I'm fine with having some content being in the blogpost. If so, the `Preparing for the New Architecture` section should be the first.